VEdger/Controllers/UserInfoController.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace VEdger.Controllers
+{
+    using System.Linq;
+    using System.Web.Mvc;
+    using VEdger.Models;
+    using VEdger.Repository;
+
+    public class UserInfoController : Controller
+    {
+
+        readonly IRepository<UserData> repository = new EntityRepository<UserData>();
+
+        [HttpGet]
+        public ActionResult Index()
+
+        {
+            var data = repository.Read<UserData>().FirstOrDefault();
+            return View(data);
+        }
+
+        [HttpPost]
+        public ActionResult Index(
+            UserData data)
+
+        {
+            var dataa = repository.Read<UserData>(data);
+            if(dataa == null)
+            {
+                repository.Create<UserData>(data);
+            }
+
+            if(dataa?.Count() == 0)
```

Comment:
co znaczy ten pytajnik bo tego nie czaje?

Reply:
Jak "dataa" jest nullem to nie wywala błędu

Reply:
https://docs.microsoft.com/pl-pl/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-